kubernetes/powerfulseal.yml

```diff
@@ -6,27 +6,21 @@ metadata:
   namespace: powerfulseal
 data:
   policy.yml: |-
-    scenarios:
-    - name: "delete a random pod in default namespace"
-      steps:
-      - matches:
-        - labels:
-            namespace: "default"
-            selector: "app=hello"
-        filters:
-        - randomSample:
-            size: 1
-        actions:
-        - kill:
-            probability: 0.77
-            force: true
+    config:
+      runStrategy:
+        minSecondsBetweenRuns: 30
+        maxSecondsBetweenRuns: 300
+    scenarios: []
```

Comment:
With `scenarios: []` container keeps restarting (`CrashLoopBackOff`). 

But, If I define a scenario here, all works as expected.

Reply:
Could you paste the error you're seeing?

Reply:
Maybe I'm misunderstanding something (I don't usually run the Seal inside k8's)

So, when I deploy Powerfulseal with this policy.yml:

```yaml
config:
    runStrategy:
      minSecondsBetweenRuns: 5
      maxSecondsBetweenRuns: 10
scenarios: []
```

Powerfulseal pod starts a never-ending loop of `CrashLoopBackOff`:

```
➜  ~ kubectl get pods --namespace=powerfulseal       
NAME                            READY   STATUS              RESTARTS   AGE
powerfulseal-79b8f869b5-nhbdh   0/1     ContainerCreating   0          11s
➜  ~ kubectl --namespace powerfulseal logs powerfulseal-79b8f869b5-nhbdh             
2020-07-08 20:08:59 INFO __main__ verbosity: None; log level: INFO; handler level: INFO
2020-07-08 20:08:59 INFO __main__ Creating kubernetes client with in-cluster config
2020-07-08 20:08:59 INFO k8s_client Initializing with config: None
2020-07-08 20:08:59 INFO __main__ No cloud driver - some functionality disabled
2020-07-08 20:08:59 INFO __main__ Using stdout metrics collector
2020-07-08 20:08:59 INFO __main__ Starting the UI server (0.0.0.0:8000)
2020-07-08 20:08:59 INFO __main__ STARTING AUTONOMOUS MODE
➜  ~ kubectl get pods --namespace=powerfulseal                       
NAME                            READY   STATUS      RESTARTS   AGE
powerfulseal-79b8f869b5-nhbdh   0/1     Completed   2          51s
➜  ~ kubectl --namespace powerfulseal logs powerfulseal-79b8f869b5-nhbdh
2020-07-08 20:09:16 INFO __main__ verbosity: None; log level: INFO; handler level: INFO
2020-07-08 20:09:16 INFO __main__ Creating kubernetes client with in-cluster config
2020-07-08 20:09:16 INFO k8s_client Initializing with config: None
2020-07-08 20:09:16 INFO __main__ No cloud driver - some functionality disabled
2020-07-08 20:09:16 INFO __main__ Using stdout metrics collector
2020-07-08 20:09:16 INFO __main__ Starting the UI server (0.0.0.0:8000)
2020-07-08 20:09:16 INFO __main__ STARTING AUTONOMOUS MODE
```

Later when I install the CDR and deploy the sandbox/scenario everything works fine. That's expected?

Reply:
This is because of this: https://github.com/bloomberg/powerfulseal/blob/32c9bd796f977b9dde7d312a76164e750c29253d/powerfulseal/policy/policy_runner.py#L82-L83

That condition doesn't make any sense now that we want to be able to run the Seal with no scenarios and then add them later. I think that it can be safely removed.

Reply:
@dnascimento would you mind removing that conditional and changing the version to `3.0.0` - I'd like to cut the release after adding this. That would be awesome. Thanks so much!

Reply:
Makes sense :ok_hand: